### PR TITLE
fixed netifly deployment issue

### LINKED ISSUE #347: [Bug]: Stop the netlify development perview on README.md changes
### Contact Details

itskanishkp.py@gmail.com

### What happened?

It take resources.

### What browsers are you seeing the problem on?

_No response_

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by ayush-129**
## [Bug]: Stop the netlify development perview on README.md changes
Fixes #347 

## Description

The issue refers to preventing Netlify from triggering a new preview deployment whenever there are changes made to the README.md file. Essentially, to configure the Netlify project to ignore changes to the README.md file, so it doesn't waste resources by redeploying the entire site when only documentation has changed. 

Screenshot 2024-10-07 204704

command in ignore will compare the previous commit (HEAD^) to the current commit (HEAD) and check if the README.md file has changed. If only the README.md was modified, Netlify will skip the build.

One more additional line change i made in footer.jsx file as twitter react icon/fa page was showing not found. I just use another twiiter icon from there. Rest everything reamins same. 

## Additional Notes
Netlify deployed link to test in mine: https://prismatic-queijadas-bc8d25.netlify.app/

### CODE REVIEW

File: public/netlify.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[build]
+publish = "build"  
+command = "npm run build" 
+ignore = "git diff --quiet HEAD^ HEAD README.md"
+
+
+[[redirects]]
```
Comment (Mr-Sunglasses) [inline public/netlify.toml:7]: Please remove the redirects as we already defined them in [public/_redirects](https://github.com/FOSS-Community/fosscu.org/blob/main/public/_redirects)

**Review by Mr-Sunglasses — COMMENTED** [review]
Thanks a lot @ayush-129 , Please check the suggestion

**Comment by ayush-129** [comment]
yup, done

**Review by Mr-Sunglasses — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**